src/azul/indexer/index_controller.py

```diff
@@ -274,6 +293,23 @@ def aggregate(self, event: Iterable[SQSRecord], *, retry=False):
             log.warning('Failed to aggregate tallies: %r', tallies_by_entity.values(), exc_info=True)
             raise
 
+    def redrive(self, event):
+        for record in event:
+            message = json.loads(record.body)
+            receive_count = record.to_dict()['attributes']['ApproximateReceiveCount']
+            attempts = message['attempts']
+            if int(receive_count) > 1:
+                attempts += 1
+            log.info('Re-driver handling message %r, attempt #%i (approx).',
+                     message, attempts)
+            if message['attempts'] == config.redrive_max_attempts:
+                log.warning('Re-driver failed to handle message %r.', message)
+                raise RedriveLambdaError(f'Worker exhausted allowed redrives for {message!r}')
+            else:
+                message['attempts'] += 1
```

Comment:
I think this will cause an infinite loop if we ever receive a message where `message['attempts']` is _greater than_ the max redrive count. Something like:
```suggestion
            if message['attempts'] < config.redrive_max_attempts:
                message['attempts'] += 1
            else:
                log.warning('Re-driver failed to handle message %r.', message)
                raise RedriveLambdaError(f'Worker exhausted allowed redrives for {message!r}')
```